### ISSUE TITLE
Command Palette key bindings are unreadable in High Contrast mode

### ISSUE THREAD

**Issue by DHowett** [microsoft/terminal#6892]
![image](https://user-images.githubusercontent.com/189190/87260417-a14b5b00-c466-11ea-8f89-8207415d6544.png)

**Comment by zadjii-msft** [comment]
Would you look at that, they sure are. Probably regressed after #6833, but hey, that's okay!

@jtippet might have an idea for the best way to fix this. I'm sure it's as easy as "make the text black in HC mode"

**Comment by jtippet** [comment]
Yeah I didn't know that UI existed, so I didn't even test it.  I've now reverse-engineered how to launch it, and can repro the bug.  It looks like the problem is that that UI element's background is using a foreground brush:

```
<!-- CommandPalette.xml -->
<Border Background="{ThemeResource SystemControlForegroundBaseLowBrush}">
  <TextBlock Foreground="{ThemeResource SystemControlForegroundBaseMediumBrush}"/>
</Border>
```

I claim that putting a foreground brush into a background attribute should generally be viewed as a datatype mismatch, or at least some sort of red flag.

If nobody beats me to it, I'll cook up a fix tonight (PST) after my day job.  I am inclined to remove the backdrop altogether when in HC mode.  That'd keep it in line with how MUX does menus:

![image](https://user-images.githubusercontent.com/10259764/87336134-3f3a3680-c4f6-11ea-8136-bf462b45376d.png)
